index.js

```diff
@@ -422,6 +422,8 @@ class Raspi extends EventEmitter {
     const pinInstance = this[getPinInstance](this.normalize(pin));
     if (pinInstance.mode === INPUT_MODE && value === HIGH) {
       this[pinMode]({ pin, mode: INPUT_MODE, pullResistor: PULL_UP });
+    } else if (pinInstance.mode === INPUT_MODE && value === LOW) {
```

Comment:
Just waiting on tests (or response of any kind) https://github.com/rwaldron/johnny-five/pull/1243